db/migrate/20221102151034_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
```

Comment:
There is a migration error when migrating the database. Kindly see the screenshot below.


![db](https://user-images.githubusercontent.com/35504149/201472073-686e3242-16ea-4611-b049-dfc0070e66c9.png)

Reply:
### Thank you @SajjadAhmad14 for reviewing my project 

I have solved the error :pray: 
[Screencast from 12.11.2022 13:52:52.webm](https://user-images.githubusercontent.com/102757126/201472748-a6c5f8c9-d607-4c87-9d50-dec1243f2eed.webm)